src/ain-db/db.ts

```diff
@@ -19,8 +23,8 @@ export default class Database {
 
   /**
    * Returns a reference instance of the given path.
-   * @param {String} path
-   * @return {Reference} A reference instance of the given path.
+   * @param {String} path The path to refer to.
+   * @returns {Reference} A reference instance of the given path.
```

Comment:
What is difference between @return and @returns?

Reply:
It seems @returns is the standard: https://typedoc.org/tags/returns/